.github/workflows/format.yaml

```diff
@@ -0,0 +1,44 @@
+name: Format
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  format:
+    if: github.repository_owner == 'astrolicious'
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Setup PNPM
+        uses: pnpm/action-setup@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18

```

Comment:
I think we want the same version as in the root package.json engines?

Reply:
my bad, will update

Reply:
there's no engines defined, nor .nvmrc

Reply:
My bad I confused it with another repo!